docs/https_source_interface.rst

```diff
@@ -4,45 +4,93 @@ HTTPS on the Source Interface
 The *Source Interface* for SecureDrop is automatically served over
 as a Tor Hidden Service, requiring a ``*.onion`` URL to access it.
 While Tor Hidden Services provide end-to-end encryption by default, as well
-as strong anonymity, high-profile organizations may wish to provide
-further verification to Sources that the SecureDrop instance belongs
-to the organization listed on the *Landing Page*.
+as strong anonymity, there are several reasons why you might want to consider
+deploying an additional layer of encryption and authentication via HTTPS:
+
+* The cryptographic primitives used by Tor Hidden Services are considered to be
+  outdated, and while there are no known compromises of the security of Tor
+  Hidden Services due to this issue, it may be wise to provide an additional
+  layer of transport encryption using stronger cryptographic primitives, which
+  is most easily achieved by setting up HTTPS on the Source Interface.
+
+  .. note:: This issue is being addressed by the Tor Project with their Next
+     Generation Onion Services design, but that design is still a work in
+     progress and is not expected to be deployed until December 2017 at the
+     soonest.
+
+* Extended Validation (EV) certificates, which are currently the only type of
+  certificates that may be issued for ``*.onion`` addresses, are intended to
+  attest to the identity of the organization running a service. This provides
+  an additional measure of authenticity (in addition to the organization's
+  *Landing Page* and the `SecureDrop Directory`_) to help assure sources that
+  they are communicating with the intended organization when they access a
+  given Source Interface.
```

Comment:
Yep, I totally agree that the Tor Onion Service crypto is weak. My concern was primarily pushing this difficult crypto decision onto users and whether it makes sense to *strongly* encourage admins to migrate over to HTTPS primarily _for crypto reasons_ at the present time given that 1) as you correctly note, the crypto is being fixed upstream and 2) imho there are other more practical attacks that can compromise sources than breaking THS crypto that we might want to focus admin energy on. 

With that said, how about we swap around these two bullets: authenticity first, crypto second. We can generally leave the discussion on Onion Service crypto in place (with one or two minor edits inline, let me know your thoughts on those). Otherwise, let's merge this in and we can revise further in a future PR if necessary.